client/package.json

```diff
@@ -65,14 +65,14 @@
 		"@types/react-router-dom": "^5.1.7",
 		"@typescript-eslint/eslint-plugin": "^5.5.0",
 		"@typescript-eslint/parser": "^5.5.0",
-		"react-scripts": "^5.0.1",
```

Comment:
We don't need this anymore since we are still using CRA (Create React App)?

Reply:
Oh, yea we still need this ... it just got moved to line 75 - https://github.com/Nonprofit-Exchange-Hub/web-app/blob/19f13606991b3f6f2e940960eba41f070fb841a9/client/package.json#L75